src/main/java/blackjack/domain/ShuffleCardsGenerator.java

```diff
@@ -0,0 +1,18 @@
+package blackjack.domain;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ShuffleCardsGenerator implements CardsGenerator {
+    @Override
+    public List<Card> generator() {
+        List<Card> cards = Arrays.stream(CardNumber.values())
+                .flatMap(cardNumber -> Arrays.stream(CardSymbol.values())
+                        .map(cardSymbol -> new Card(cardNumber, cardSymbol))).collect(
+                        Collectors.toList());
```

Comment:
```suggestion
        List<Card> cards = Arrays.stream(CardNumber.values())
                .flatMap(cardNumber -> Arrays.stream(CardSymbol.values())
                        .map(cardSymbol -> new Card(cardNumber, cardSymbol)))
                .collect(Collectors.toList());
```
스트림에서 연산자는 개행해주는 것이 가독성에 도움이 됩니다!

Reply:
분명히 연산자를 기준으로 개행을 해주었다고 생각했는데 놓친 부분인것 같습니다 🥲 수정하였습니다!